File: generative-ai/2-doc-examples/2-3-promptfoo/3-prompts-tests-outputs/1-prompts/prompts/prompt.py
def prompt_func(context: dict) -> str:
    variables: dict = context['vars']

    return [
        {
            "role": 'system',
            "content": "You're a translator. Your task is translate user input to French.",
        },
        {
            "role": 'user',
            "content": f"{variables['text']}",
        },
    ]

File: generative-ai/llm/2-doc-examples/2-3-promptfoo/3-prompts-tests-outputs/1-prompts/prompts/prompt.js
module.exports.promptFunc = async function ({ vars }) {
    return [
      {
        role: 'system',
        content: `You're a translator. Your task is translate your input to French.`,
      },
      {
        role: 'user',
        content: `${vars.text}`,
      },
    ];
  };